### PR TITLE
cmdline iframe css refactor

### PR DESCRIPTION
**Pull request by maoanran**
@cmcaine

### CODE REVIEW

File: src/commandline_content.ts [hunk 1]
```diff
@@ -34,21 +35,17 @@ document.addEventListener("DOMContentLoaded", init)
 // This second call will fail in the most common case, but makes web-ext reloads effective.
 init()
 
-// The CSS in this file should probably go in static/
 export function show(){
-    // I don't understand how CSS works - but this ensures that the commandline is always at the bottom of the page.
-    cmdline_iframe.setAttribute("style", "position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: 24px; border: 0; padding: 0; margin: 0;");
```
Comment (maoanran) [inline src/commandline_content.ts]: I just remove this and don't find any issue, so how can this issue recur? :)

File: src/commandline_content.ts [hunk 2]
```diff
@@ -34,21 +35,17 @@ document.addEventListener("DOMContentLoaded", init)
...
     const height = cmdline_iframe.contentWindow.document.body.offsetHeight + 'px'
-    cmdline_iframe.setAttribute("style", `position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: ${height}; border: 0; padding: 0; margin: 0;`);
+    cmdline_iframe.setAttribute("style", `height: ${height};`);
 }
 
 export function resize() {
-    const height = cmdline_iframe.contentWindow.document.body.offsetHeight + 'px'
-    cmdline_iframe.setAttribute("style", `position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: ${height}; border: 0; padding: 0; margin: 0;`);
+    show()
```
Comment (maoanran) [inline src/commandline_content.ts]: Why do we need this resize? looks like same with show.

File: src/commandline_content.ts (hunk 1 above)
Comment (bovine3dom) [reply src/commandline_content.ts]: I can't reproduce the issue either on current master.

It is still reproducible at 242fcce, though, where commenting out that line will stop the command line from showing. Something somewhere else must have fixed the underlying issue.

File: src/commandline_content.ts (hunk 2 above)
Comment (bovine3dom) [reply src/commandline_content.ts]: It is the same - we don't need it any more.

File: src/commandline_content.ts [hunk 3]
```diff
@@ -34,21 +35,17 @@ document.addEventListener("DOMContentLoaded", init)
 // This second call will fail in the most common case, but makes web-ext reloads effective.
 init()
 
-// The CSS in this file should probably go in static/
 export function show(){
-    // I don't understand how CSS works - but this ensures that the commandline is always at the bottom of the page.
-    cmdline_iframe.setAttribute("style", "position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: 24px; border: 0; padding: 0; margin: 0;");
     const height = cmdline_iframe.contentWindow.document.body.offsetHeight + 'px'
-    cmdline_iframe.setAttribute("style", `position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: ${height}; border: 0; padding: 0; margin: 0;`);
+    cmdline_iframe.setAttribute("style", `height: ${height};`);
```
Comment (bovine3dom) [inline src/commandline_content.ts]: Just a little point on style - we aren't using semi-colons at the end of lines. I'm not sure why there was one in the original code.

**Comment by bovine3dom** [comment]
I believe you have introduced a bug where pressing ESC to close the command line does not cause it to lose focus.

Steps to reproduce:

1. Press `:` to open the command line.
2. Press `Escape` to close it.
3. Try scrolling around with j/k, opening command line with `:`, etc. (It shouldn't work).
4. Click on the page to focus it.
5. Press `:` to open the command line and observe all the j/k etc that you've typed.

I haven't yet been able to pinpoint what is causing it.

**Comment by maoanran** [comment]
@bovine3dom HI, thanks for review, do you mean this PR introduce the issue, it seems works after rebase. :)

**Comment by bovine3dom** [comment]
Yes, it does work after rebase. When I did `hub pull (your PR)` it had the issue, so I assumed it was your fault. If you rebase it, I'll merge the PR.

**Pull request merged**